tests/Doctrine/Tests/OrmFunctionalTestCase.php

```diff
@@ -328,6 +331,11 @@
             Models\Issue5989\Issue5989Employee::class,
             Models\Issue5989\Issue5989Manager::class,
         ],
+        'gh7505' => [
```

Comment:
Since this is for just one ticket, consider squashing the entire model into the ticket test case class, as it is done with other similar tests